routers/private/hook_post_receive.go

```diff
@@ -195,10 +195,17 @@ func HookPostReceive(ctx *gitea_context.PrivateContext) {
 
 			branch := refFullName.BranchName()
 
-			// If our branch is the default branch of an unforked repo - there's no PR to create or refer to
-			if !repo.IsFork && branch == baseRepo.DefaultBranch {
-				results = append(results, private.HookPostReceiveBranchResult{})
-				continue
+			if branch == baseRepo.DefaultBranch {
+				err := repo_module.UpdateRepoLicensesByGitRepo(ctx, repo, nil)
+				if err != nil {
+					ctx.JSON(http.StatusInternalServerError, private.Response{Err: err.Error()})
+				}
```

Comment:
No `return`?

Reply:
Done in https://github.com/go-gitea/gitea/pull/24872/commits/44cf8e4eee10a7759ffbd8cc40b210d83d934e70.